backend/app/routers/groups.py

```diff
@@ -34,6 +35,7 @@ async def get_groups(
     user_id=Depends(get_user),
     skip: int = 0,
     limit: int = 10,
+    superadmin: bool = False,
```

Comment:
Is this flag being used in the logic?

Reply:
yes, it is used in `get_admin_mode` function above. this is a param we can add to any function that has `Depends(get_admin_mode)`